doc/api/path.md

```diff
@@ -83,6 +84,8 @@ and is not a string.
 added: v0.9.3
 -->
 
+* {String}
```

Comment:
On the other hand, people seem to be fine with it in https://github.com/nodejs/node/pull/9505, so I guess it's okay your way.